rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/RMRestInterface.java

```diff
@@ -177,6 +177,24 @@ RMStateDelta getRMStateDelta(@HeaderParam("sessionid") String sessionId,
     RMStateFull getRMStateFull(@HeaderParam("sessionid") String sessionId)
             throws NotConnectedException, PermissionRestException;
 
+    /**
+     * Returns a task variable model string containing the list of registered hosts in the resource manager
+     * @return a model containing the list of hosts, including an empty name
+     */
+    @GET
+    @Path("model/hosts")
+    @Produces("text/plain")
+    String getModelHosts() throws PermissionRestException;
```

Comment:
Could you add an exmple in the javadoc of the format. As it is a text/plain format it is really not clear what will be the response.

Reply:
ok